src/typescript/frontend/src/components/pages/home/components/emoji-table/index.tsx

```diff
@@ -66,18 +66,13 @@ const EmojiTable = (props: EmojiTableProps) => {
   }, [searchBytes]);
 
   const pushURL = useEvent((args?: { page?: number; sort?: SortMarketsBy; emojis?: string[] }) => {
-    const curr = new URLSearchParams(location.search);
     const newURL = constructURLForHomePage({
       page: args?.page ?? page,
       sort: args?.sort ?? sort,
       searchBytes: encodeEmojis(args?.emojis ?? emojis),
     });
 
-    // Always push the new URL to the history, but only refresh if the URL has actually changed in a meaningful way.
     router.push(newURL.toString(), { scroll: false });
-    if (isHomePageURLDifferent(curr, newURL.searchParams)) {
-      router.refresh();
-    }
```

Comment:
This isn't doing nothing- `router.refresh()` triggers the server components to refetch their data. If you only do `router.push`, it will only change the URL in the browser's URL bar, but not actually reload the page.

The `isHomePageURLDifferent` is there to avoid a refresh when the home page is functionally the same (e.g. `/home` and `/` and `/home?sort_by=market_cap` are all the same page)

Reply:
Yes I checked to make sure it works still- I think I possibly added this while confused about how the router cache worked since we'd recently updated it to use a time of `0` instead of `30`, plus it may have also been bugged back then, too.

I checked if it sees new markets without the `router.refresh()` just by using push and it does, so you're right, all's good!

I agree about unused code- in which case I'll also remove `isHomePageURLDifferent` since it's only used in one place